### ISSUE TITLE
During-Protest room tone, BP

### ISSUE THREAD

**Issue by scottpeterson** [gryphonshafer/Quizzing-Rule-Book#33]
Challenges and Protests are important tools to ensure the rules get followed, and ensure quiz participants have direct input.

A downside of Challenges and especially Protests is that, when they are concluded, the mood/tone of the room has gotten significantly worse. This often occurs, regardless of the disagreement level or contentiousness level of the Challenge/Protest.

As such, the norm should be introduced, and encouraged by quiz officials, coaches, and others, that during Protests, while the officials and coaches are elsewhere, quizzer should be encouraged to leave the stage, converse with each other and the audience, and more.

**Comment by ARMediting** [comment]
I agree about the tone of the room, but opening the door to a single scenario where quizzers are allowed to interact closely with the audience during a quiz--without oversight, no less--seems unusual. However, looking again at the rulebook, nothing in the Platform Protocol section forbids audience communication between questions. Perhaps the rule to include then is that all copies of the quiz questions are either taken with the officials and coaches or locked on a device.

**Comment by ZacharyTinker** [comment]
I think allowing the quizzers freedom to interact with those still in the room is good. 

Outlining protest protocol is probably what is needed most.

1) A protest is announced
2) Quiz master briefly explains to the room what will happen
3) All officials take their materials out of the room (or lock their computer if online)
4) Officials and coaches leave the room
5) Discussion ensues

**Comment by josiah-leinbach** [comment]
@ZacharyTinker is right that the circumstances surrounding protests has contributed to their negative connotation, not the other way around. This and the fact that the word "protest" already carries a heavy, serious tone. Having been in several such situations, I completely agree this is a problem.

Therefore, I think we should amend the last sentence of 4.4 to say:

> The quiz officials and quiz team coaches will meet privately and attempt to come to an agreement. If no agreement is reached within 10 minutes, the meet director will arbitrate and determine the just solution. _During the protest, quizzers are permitted to leave their seats and converse with other quizzers as well as audience members._

Then after this should be added the following sentence in a new paragraph:

> After a coach announces a protest, the quizmaster shall inform the room about the procedure outlined in the preceding paragraph.

**Comment by jswingle** [comment]
I like @josiah-leinbach 's wording. And I like the general point of the rule change -- there isn't any reason for quizzers to be quiet during a protest. Once the protest has been announced, no further input from the quizzers can affect the outcome of that particular quiz question.

**Comment by josiah-leinbach** [comment]
I will make an addendum to my proposed wording from above:

Calling it a "protest", a word imbued with negative connotations, creates the negative atmosphere we are trying to avoid. Many "protests" are just seeking to uphold the rules of a quiz or clarify that a potential violation. For example, if the quizmaster is asking Question 20B as a CVR but the maximum has already been exceeded, a coach who says, "Wait. I think we've exceeded the maximums on References already," is protesting the quiz in a logistical capacity. To call that a "protest" seems a bit too strong.

Furthermore, many protests take place at crucial moments in a quiz or tournament, so tensions already run high. Compound this with the fact that the word "protest" carries a personal tone, further raising those tensions, and 4.4 speaks about the coach "feeling" that something incorrect has taken place. 

Changing the language surrounding protests and calling them something else would go a long way to making them less contentious. I think "review" is a good substitute, but others are fine too. So in addition to my prior wording, I would modify all of 4.4 to accommodate this (besides the needed changes in 2.2 and 5.1.3). The whole revision would be as follows, with all changes (including my prior wording) bolded and italicized:

**4.4**
> Coaches, including assistant coaches, of the teams participating in a quiz may place the quiz under **_review_** under the following conditions:
> 
> - When a coach **_thinks_** a logistical rule has been broken by the quiz officials
> - When a coach **_thinks_** a ruling on a challenge is incorrect
>
> The **_review_** must apply to the question or ruling that just occurred. 
>
>The **_review_** must be placed before the quizmaster makes known the next question's type. If a **_review_** is **_initiated_** after question number 20 is completed it must be done immediately after the ruling is announced.
> 
> The coach may only confer with their assistant coach and any quizzer members of the team that are not currently seated in the quiz before _**initiating a review**_.
> 
> The quiz officials and quiz team coaches will meet privately and attempt to come to an agreement. If no agreement is reached within 10 minutes, the meet director will arbitrate and determine the just solution. **_During the protest, quizzers are permitted to leave their seats and converse with other quizzers as well as audience members._**
>
> **_After a coach announces a protest, the quizmaster shall inform the room about the procedure outlined in the preceding paragraph._**

**2.2**
> If the additional information regarding the ruling is provided at the discretion of the QM/AJ and not at the request of the captain, this is not considered an explanation of a ruling, which would invalidate a challenge or **_review_**.
> 
> If the captain requests additional information regarding the ruling, this is considered an explanation of a ruling and invalidates a challenge or **_review_**. Before replying to a request for an explanation, the QM must ask all teams if they wish to challenge.

**5.1.3**

> -10 points for 2nd and subsequent overruled challenges and **_reviews_**

**Comment by gryphonshafer** [comment]
I'm fully on board with changing "protest" to "review" for the reasons Josiah states: it's a more accurate word for the intent, nature, and purpose of the thing. For the same reasons, I'd like to see "challenge" changed as well. Inexhaustive list of possible options:

- Appeal
- Objection
- Petition

**Comment by jttower** [comment]
I agree with changing "protest" to "review" as outlined above.  There is one instance of "protest" missed in Josiah's wording above, in the last paragraph of 4.4, otherwise I think it looks great.

**Comment by ARMediting** [comment]
Also, I like "appeal" instead of "challenge," but I'd be curious to get input from a variety of current quizzers on what they like. Should we put this in a different issue?

**Comment by ARMediting** [comment]
Now my response to @josiah-leinbach proposed wording: Instead of "initiating a review," should it "calling for a review"? I don't want to give QMs power to arbitrarily ignore protests, but there are a few occasions where a protest can't be made (such as if an requested explanation was just given). Same goes for "After a coach announces a protest,"—would this be better as "if a review is initiated," ie, approved by the QM after a coach calls for it?

**Comment by josiah-leinbach** [comment]
@ARMediting I agree with your proposed modifications.

**Comment by josiah-leinbach** [comment]
On September 11th, 2022, the Rules Committee voted 5-0 with one absence to adopt the following change to section 4.4:

> Coaches, including assistant coaches, of the teams participating in a quiz may call for a review under the following conditions:
> 
> - When a coach thinks a logistical rule has been broken by the quiz officials
> - When a coach thinks a ruling on a challenge is incorrect
> 
> The review must apply to the question or ruling that just occurred.
> 
> The review must be called before the quizmaster makes known the next question's type. If a review is called after question number 20 is completed it must be done immediately after the ruling is announced.
> 
> The coach may only confer with their assistant coach and any members of the team that are not currently seated in the quiz before calling for a review.
> 
> The quiz officials and quiz team coaches will meet privately and attempt to come to an agreement. If no agreement is reached within 10 minutes, the meet director will arbitrate and determine the just solution. During the review, quizzers are permitted to leave their seats and converse with other quizzers as well as audience members.
> 
> After a review is called, the quizmaster shall inform the room about the procedure outlined in the preceding paragraph.

Besides this, all instances elsewhere of “protest” would change to “review".
